package.json

```diff
@@ -39,7 +43,10 @@
     "test:coverage": "jest --coverage",
     "test:debug": "node --nolazy --inspect-brk ./node_modules/.bin/jest --runInBand --no-cache",
     "test:watch": "jest --watchAll",
-    "test:coverage:watch": "jest --coverage --watchAll"
+    "test:coverage:watch": "jest --coverage --watchAll",
+    "cypress:test": "cypress run",
+    "cypress:open": "cypress open",
```

Comment:
My inclination would be to call these `test:cypress` and `test:cypress:open`(?) so that all the testing related scripts start with `test`.

Reply:
I keep messing that up actually, so I agree with this